main.py

```
@@ -38,6 +39,13 @@ async def load_model_async():
 async def startup_event():
     app.state.model = await load_model_async()
 
+@app.get("/ai/health")
+def health_check():
```

Ruff (0.11.9)

Missing return type annotation for public function `health_check`

(ANN201)